src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -730,6 +719,11 @@
     <Compile Include="System\Xml\Xsl\Xslt\XslVisitor.cs" />
     <Compile Include="$(CommonPath)\System\LocalAppContext.cs" />
     <Compile Include="System\Xml\Core\LocalAppContextSwitches.cs" />
+    <Compile Include="System\CodeDom\CodeDomProvider.cs" />
+    <Compile Include="System\CodeDom\CodeGenerator.cs" />
+    <Compile Include="System\CodeDom\CompilerError.cs" />
+    <Compile Include="System\CodeDom\CompilerResults.cs" />
```

Comment:
These classes are used in few places, like [XslCompiledTransform](https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/src/System/Xml/Xslt/XslCompiledTransform.cs), [Compiler ](https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/src/System/Xml/Xsl/Xslt/Compiler.cs), etc. for error reporting.